### PR TITLE
Therapy groups fixes

### PR DESCRIPTION
**Pull request by eyalvo-matrix**
Some groups hot fixes to adapt to the new design

### CODE REVIEW

File: sites/default/sqlconf.php [hunk 1]
```diff
@@ -28,3 +28,4 @@
 //////////////////////////
 //////////////////////////
 //////////////////////////
+?>
```
Comment (bradymiller) [inline sites/default/sqlconf.php]: Revert these changes by copying over sqlconf.php from master. Also do the same with application.config.php above.

File: sql/5_0_0-to-5_0_1_upgrade.sql [hunk 2]
```diff
@@ -669,4 +669,8 @@ UPDATE `openemr_postcalendar_categories` SET pc_constant_id = LOWER(REPLACE (pc_
 
 #IfMissingColumn facility facility_taxonomy
 ALTER TABLE facility ADD facility_taxonomy VARCHAR(15) DEFAULT NULL;
-#EndIf
\ No newline at end of file
+#EndIf
+
+#IfTable form_therapy_groups_attendance
```
Comment (bradymiller) [inline sql/5_0_0-to-5_0_1_upgrade.sql]: Very nice to see your thinking about this!

Comment (bradymiller) [reply sql/5_0_0-to-5_0_1_upgrade.sql]: However, to make this bulletproof, place this above the code block you modified at line 258. This then means we won't be able to break it even if run the upgrade script multiple times.

**Comment by eyalvo-matrix** [comment]
Thanks for the comments , I applied the changes

File: sites/default/sqlconf.php (hunk 1 above)
Comment (eyalvo-matrix) [reply sites/default/sqlconf.php]: of course this was done unintentionally

File: sql/5_0_0-to-5_0_1_upgrade.sql (hunk 2 above)
Comment (eyalvo-matrix) [reply sql/5_0_0-to-5_0_1_upgrade.sql]: sure :+1:

**Comment by bradymiller** [comment]
Code looks great. Going to bring this into the codebase. Thanks for the fixes and Happy Holidays! -brady
